reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -112,10 +122,7 @@
     "%%time\n",
     "\n",
     "# run in local mode?\n",
-    "local_mode = False\n",
-    "\n",
-    "image = '462105765813.dkr.ecr.{}.amazonaws.com/sagemaker-rl-coach-container:coach-1.0.0-tf-cpu-py3'.format(region_name)\n",
```

Comment:
I am curious how many images do we have?

Reply:
This repo has a list of all sagemaker images: https://github.com/aws/sagemaker-rl-container